### ISSUE TITLE
Why all my predicted labels are all 'O' ?

### ISSUE THREAD

**Issue by CherryCola0107** [Chaarangan/NERP#31]
I printed the predicted labels of the models and found out no matter how I change the pre-trained model, the predicted results are all 'O'. Here is how I printed and the format of my training data.
![image](https://github.com/user-attachments/assets/2ec098f5-dffb-4d63-aa4d-112fbe88aae2)
![image](https://github.com/user-attachments/assets/98b0142f-c193-4293-9b7e-8b5567949b25)
Is my format of dataset with some problems or?  Thank you!
